locksmith/src/utils/middlewares/auth.ts

```diff
@@ -94,6 +95,7 @@ export const authMiddleware: RequestHandler = async (req, _, next) => {
       return next()
     }
     if (tokenType === 'bearer') {
+      // TODO: accept privy access tokens too
```

Comment:
this seems to be the main area where locksmith will determine what tokens to "permit". in the process of mapping out the best way to achieve that and still support legacy ones

Reply:
We don't need this either